controller/contour/rbac.go

```diff
@@ -4,10 +4,10 @@ import (
 	"context"
 	"fmt"
 
-	projectcontourv1 "github.com/projectcontour/contour/apis/projectcontour/v1"
 	operatorv1alpha1 "github.com/projectcontour/contour-operator/api/v1alpha1"
 	oputil "github.com/projectcontour/contour-operator/util"
 
+	projectcontourv1 "github.com/projectcontour/contour/apis/projectcontour/v1"
```

Comment:
Did you mean to move it after the blank line?  `go fmt` re-orders but does not re-group on my system:
```diff
-	projectcontourv1 "github.com/projectcontour/contour/apis/projectcontour/v1"
 	operatorv1alpha1 "github.com/projectcontour/contour-operator/api/v1alpha1"
 	oputil "github.com/projectcontour/contour-operator/util"
+	projectcontourv1 "github.com/projectcontour/contour/apis/projectcontour/v1"
 ```